### PR TITLE
KS-TRANSLATIONS-DE-ES-IT-PL-PT-Dedicated-Server-Uninstall-RTMv2

### PR DESCRIPTION
**Pull request by Kazarus4**
All translations are done (DE, ES-ES, ES-US, IT, PL and PT.

Need proof of these translations before merge.

### CODE REVIEW

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/translator.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.translations = yaml.load(fd)
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/banner.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.banners = yaml.load(fd)
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 303, in run
    tabs = self.build_tabs_object(lines)
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 280, in build_tabs_object
    if line[0] != ">":
WARNING: Empty alt attribute for image 1149.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/microsoft-collaborative-solutions/how_to_configure_android/guide.pt-pt.md
WARNING: Empty alt attribute for image 1150.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/microsoft-collaborative-solutions/how_to_configure_android/guide.pt-pt.md
WARNING: Empty alt attribute for image 1151.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/microsoft-collaborative-solutions/how_to_configure_android/guide.pt-pt.md
WARNING: Empty alt attribute for image 1152.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/microsoft-collaborative-solutions/how_to_configure_android/guide.pt-pt.md
WARNING: Other images have empty alt attributes
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 303, in run
    tabs = self.build_tabs_object(lines)
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 280, in build_tabs_object
    if line[0] != ">":
ERROR: Could not process /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/cloud/dedicated/rtm-uninstall/guide.pl-pl.md
  | ScannerError: while scanning a simple key
  in "<unicode string>", line 2, column 1:
    excerpt:"Dowiedz się, jak odinst ... 
    ^
could not find expected ':'
  in "<unicode string>", line 3, column 1:
    updated: 2023-06-20
    ^
CRITICAL: RuntimeError: Error encountered
```

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/translator.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.translations = yaml.load(fd)
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/banner.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.banners = yaml.load(fd)
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 303, in run
    tabs = self.build_tabs_object(lines)
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 280, in build_tabs_object
    if line[0] != ">":
WARNING: Empty alt attribute for image 1149.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/microsoft-collaborative-solutions/how_to_configure_android/guide.pt-pt.md
WARNING: Empty alt attribute for image 1150.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/microsoft-collaborative-solutions/how_to_configure_android/guide.pt-pt.md
WARNING: Empty alt attribute for image 1151.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/microsoft-collaborative-solutions/how_to_configure_android/guide.pt-pt.md
WARNING: Empty alt attribute for image 1152.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/microsoft-collaborative-solutions/how_to_configure_android/guide.pt-pt.md
WARNING: Other images have empty alt attributes
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 303, in run
    tabs = self.build_tabs_object(lines)
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 280, in build_tabs_object
    if line[0] != ">":
ERROR: Skipping /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/cloud/dedicated/rtm-uninstall/guide.pt-pt.md: could not find information about 'title'
CRITICAL: RuntimeError: Error encountered
```

File: pages/cloud/dedicated/rtm-uninstall/guide.pt-pt.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+título: "Desinstalar o sistema de monitorização RTM v2"
```
Comment (OVH-Team-Guides) [inline pages/cloud/dedicated/rtm-uninstall/guide.pt-pt.md]:
```suggestion
title: "Desinstalar o sistema de monitorização RTM v2"
```

**Comment by OVH-Team-Guides** [comment]
@tcpdumpfbacke , please check DE version

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**